tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -149,63 +158,101 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             sequence.push({
-                event: "{trackTooltips}.events.notifyFocusChange",
-                listener: function () {
-                    fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
-                        jqUnit.assertNode("The contents of the tooltip should be set", tooltipContents[0], $("b", tooltip));
-                    });
-                }
+                func: "fluid.changeElementValue",
+                args: ["pager.dom.pageSize", pageSize]
             });
-        };
-
-        var tooltipTest = function (location) {
-            return function (idx, linkEl) {
-                var linkId = linkEl.id;
-                var link = $(linkEl);
-                sequence.push({
-                    element: link,
-                    jQueryTrigger: "focus"
-                });
-
-                if (link.hasClass(pager.pagerBar.options.styles.currentPage)) { // assumption that this will not change between now and then!
-                    assertVisibleTips("There shouldn't be any tooltips visible when the currentPage is focused", []);
-                } else {
-                    assertVisibleTips("Only the tooltip for page link " + (idx + 1) + ", in the " + location + " page bar is visible", [linkId]);
-                }
-                sequence.push({
-                    element: link,
-                    jQueryTrigger: "blur"
-                });
-                assertVisibleTips("There shouldn't be any tooltips visible when none of the pageLinks are focused", []);
-            };
-        };
+            sequence.push({
+                func: "fluid.tests.verifyLinksAndTooltips",
+                args: [pager, pageSize, linksList]
+            });
+        }
 
-        pageLinksTop.each(tooltipTest("top"));
-        pageLinksBottom.each(tooltipTest("bottom"));
         return {
             name: "Pager tooltip tests",
-            tests: {
-                name: "Tooltip visibility",
-                sequence: sequence
-            }
+            tests: [
+                {
+                    name: "Tooltip visibility and contents",
+                    sequence: sequence
+                }
+            ]
         };
     };
 
+    fluid.tests.getTooltipContents = function(listOfData, pageSize) {
+        var tooltipContents = [];
+        for (var i = 0; i < listOfData.length; i += pageSize) {
+            var tooltipContent = [];
+            var pageStartIndex = i;
+            var pageEndIndex = Math.min(pageStartIndex + pageSize -1, listOfData.length - 1);
+            tooltipContent.push({nodeName: "b", nodeText: listOfData[pageStartIndex].text});
+            tooltipContent.push({nodeName: "b", nodeText: listOfData[pageEndIndex].text});
+            tooltipContents.push(tooltipContent);
+        }
+
+        return tooltipContents;
+    };
+
+    fluid.tests.verifyLinksAndTooltips = function(pager, pageSize, linksList) {
```

Comment:
whitespace issue. please make sure you define functions like `some.namesapce = function (arg1, arg2) {};` _**Note:** the space between the "function" and "("_  Please make sure to fix this up throughout.